qt/applications/workbench/workbench/plotting/figureinteraction.py

```diff
@@ -136,6 +136,9 @@ def on_scroll(self, event):
             and len(event.inaxes.lines) == 0
         ):
             return
+
+        self.correct_for_scroll_event_on_legend(event)
```

Comment:
```suggestion
        self._correct_for_scroll_event_on_legend(event)
```
Put underscore at beginning of name to signify that function is only used in this file